websites/docs-smoke-test/gatsby-config.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
           resolve: '@commercetools-docs/gatsby-theme-learning',
           options: {
             auth0Domain: process.env.GATSBY_AUTH0_DOMAIN,
-            learnApiDomain: process.env.GATSBY_LEARNAPI_DOMAIN,
+            learnApiDomain: process.env.GATSBY_LEARN_API_DOMAIN,
```

Comment:
@gabriele-ct  I fixed this typo in this PR ot have the variable match the one in the GitHub actions secrets.  But I have the slight feeling it could be the reason that the build is failing.